docs/release-notes/NuGet-6.10.md

```diff
@@ -0,0 +1,178 @@
+---
+title: NuGet 6.10 Release Notes
+description: Release notes for NuGet 6.10 including new features, bug fixes, and DCRs.
+author: kartheekp-ms
+ms.author: kartheekp-ms
+ms.date: 5/13/2024
+ms.topic: conceptual
+---
+
+# NuGet 6.10 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.10**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.10](https://visualstudio.microsoft.com/downloads/) | [8.0.3xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.10
+
+* Add event tracing to restore to make it possible to measure performance - [#13274](https://github.com/NuGet/Home/issues/13274)
+
+* Warn when vulnerabilities are detected during packages.config restore in CLI scenarios. - [#13253](https://github.com/NuGet/Home/issues/13253)
```

Comment:
Let's move this to issues fixed, since https://github.com/NuGet/Home/issues/12307 is the parent issue.

Reply:
Fixed in the latest commit.